gene2phenotype_project/gene2phenotype_app/serializers/user.py

```diff
@@ -259,7 +302,7 @@ def create(self, validated_data):
             panel_obj = Panel.objects.get(name=panel)
 
             if UserPanel.objects.filter(user=user_obj, panel=panel_obj, is_deleted=1).exists():
```

Comment:
When I try to add a user-panel for a row that is defined as `is_deleted=1`, I'm getting the message: `"User ... already exists in this ..."`
This happens because after this line, the code goes to line 307 and fetches the row that was just updated. There is an else missing.

Reply:
Updated this but because panel is a list field, I did it a bit differently. Please let me know if it addresses the issues